src/internal/pbutil/pbutil.go

```diff
@@ -93,3 +96,31 @@ func NewWriter(w io.Writer) Writer {
 func NewReadWriter(rw io.ReadWriter) ReadWriter {
 	return &readWriter{r: rw, w: rw}
 }
+
+func SanitizeTimestampPb(timestamp *timestamppb.Timestamp) time.Time {
+	if timestamp != nil {
```

Comment:
So you have 4 new functions here, 2 of which checks for non-nil first, but the other two checks for nil first. I prefer checking for nil condition first. But more importantly, lets just try to be consistent.

```suggestion
	if timestamp == nil {
	    return time.Time{}
	}
```

Reply:
Makes sense!